boards/arm/nrf52832_mdk/Kconfig.defconfig

```diff
@@ -16,6 +16,13 @@ config I2C_0
 
 endif # I2C
 
+if PWM
+
+config PWM_0
```

Comment:
we should start moving to the more compact convention:

```
config PWM_0
  default PWM
```

Like with Bluetooth below